pages/interfaces/VerifiedPartner.ts

```diff
@@ -0,0 +1,66 @@
+export interface VerifiedPartnerResponse {
+  data: VerifiedPartner[]
+}
+
+export interface VerifiedPartner {
+  id: number
+  date_created: string
+  date_updated: string
+  name: string
+  description: string
+  logo: string
+  team_size: TeamSize
+  services: Service[]
+  region: Region
+  country: string
+  languages: string[]
+  payment_methods: PaymentMethod[]
+  website: null | string
+  youtube: null | string
+  email: null | string
+  twitter: null | string
+  instagram: null | string
+  linkedin: null | string
+  discord: null | string
+  opensea: null | string
+  sketchfab: null | string
+  parcel: null | string
+  marketplace: null | string
+  status: Status
+  user: string
+}
+
+export enum PaymentMethod {
+  CreditCard = "Credit Card",
```

Comment:
Are all these enum mappings just to show texts for the UI? It'd be great to add them to the locale file.

Reply:
These Enums come from the API, so it seems to me that it does not make sense to add them to the locale 🤔